server/app/assets/javascripts/azure_upload.ts

```diff
@@ -36,6 +36,11 @@ class AzureUploadController {
       throw new Error('Attempted to upload to null container')
     }
     const azureUploadProps = this.getAzureUploadProps(uploadContainer)
+    if (azureUploadProps.file == null) {
+      // No file selected by the user. Validation is done in file_upload.ts so
```

Comment:
Does the validation also including error messaging for the user along the lines of "no file selected, please choose a file befores starting an upload."?

Reply:
Yes, it does. it shows a hidden div containing error message.